rpm/sailfish-secrets.spec

```diff
@@ -96,6 +96,7 @@ BuildRequires:  qt5-tools
 Summary:    Unit tests for the libsailfishcrypto library.
 Group:      System/Libraries
 BuildRequires: pkgconfig(Qt5QuickTest)
+BuildRequires: pkgconfig(libcrypto)
```

Comment:
Is this a requirement for the tests?  (is this in order to build the openssl test plugin, or?)

Reply:
Yes, it's for openssl test plugin, the testopensslcrypto .o contains call to RAND_add symbol for instance. So it must link with libcrypto. Though RAND_add is not found in the source file. I guess it's coming from inline functions from the header?